<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>873</width>
    <height>593</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>AnkiFlash</string>
  </property>
  <property name="windowIcon">
   <iconset theme="anki.png">
    <normaloff>.</normaloff>.</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QGroupBox" name="inputBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>421</width>
     <height>441</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>Input</string>
   </property>
   <widget class="QPlainTextEdit" name="inputTxt">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>381</width>
      <height>161</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="placeholderText">
     <string>Input your words</string>
    </property>
   </widget>
   <widget class="QPushButton" name="generateBtn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>390</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="toolTip">
     <string>Generate flashcards for the input words</string>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Generate</string>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="totalLbl">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>195</y>
      <width>200</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Total: 0</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="sourceBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>224</y>
      <width>98</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="title">
     <string>Source</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QRadioButton" name="japaneseSource">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>56</y>
       <width>78</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
       <strikeout>false</strikeout>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="text">
      <string>Japanese</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="vietnameseSource">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>74</y>
       <width>78</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
       <strikeout>false</strikeout>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="text">
      <string>Vietnamese</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="frenchSource">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>38</y>
       <width>78</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
       <strikeout>false</strikeout>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="text">
      <string>French</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="englishSource">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>78</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
       <strikeout>false</strikeout>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="text">
      <string>English</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="translatedToBox">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>224</y>
      <width>161</width>
      <height>151</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Translated To</string>
    </property>
    <widget class="QRadioButton" name="chineseSimpTarget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Chinese (Simplified)</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="vietnameseTarget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>42</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Vietnamese</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="englishTarget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>23</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>English</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="japaneseTarget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>81</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Japanese</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="chineseTradTarget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Chinese (Traditional)</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="frenchTarget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>62</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>French</string>
     </property>
    </widget>
   </widget>
   <widget class="QProgressBar" name="generateProgressBar">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>390</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="value">
     <number>1</number>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="outputBox">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>10</y>
     <width>421</width>
     <height>441</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>Output</string>
   </property>
   <widget class="QPlainTextEdit" name="outputTxt">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>381</width>
      <height>161</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="placeholderText">
     <string>Output cards</string>
    </property>
   </widget>
   <widget class="QLabel" name="completedLbl">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>196</y>
      <width>200</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Completed: 0</string>
    </property>
   </widget>
   <widget class="QLabel" name="failureLbl">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>410</y>
      <width>200</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Failure: 0</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="failureTxt">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>381</width>
      <height>161</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="placeholderText">
     <string>Failure words</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="importerBox">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>460</y>
     <width>421</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>Importer</string>
   </property>
   <widget class="QProgressBar" name="importProgressBar">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>70</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="value">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="deckNameLbl">
    <property name="geometry">
     <rect>
      <x>23</x>
      <y>37</y>
      <width>63</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Deck Name</string>
    </property>
   </widget>
   <widget class="QPushButton" name="importBtn">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="toolTip">
     <string>Import generated flashcards into Anki</string>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Import</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="deckNameTxt">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Input deck name</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="ankiflashInfoBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>460</y>
     <width>421</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>AnkiFlash Info</string>
   </property>
   <widget class="QLabel" name="buycoffee">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>376</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>If you want to buy me a coffee: &lt;a href=&quot;https://www.buymeacoffee.com/longhoangly&quot; target=&quot;_blank&quot;&gt;https://www.buymeacoffee.com/longhoangly&lt;/a&gt;</string>
    </property>
    <property name="openExternalLinks">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="inbox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>298</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Feel free to drop me a message if any issue or any suggestion</string>
    </property>
   </widget>
   <widget class="QLabel" name="caption">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>284</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Sans Serif</family>
      <weight>50</weight>
      <italic>true</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Always beside and help you to learn vocabularies</string>
    </property>
   </widget>
   <widget class="QLabel" name="support">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>272</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Support: &lt;a href=&quot;https://www.facebook.com/ankiflashcom/inbox&quot; target=&quot;_blank&quot;&gt;https://www.facebook.com/ankiflashcom/inbox&lt;/a&gt;</string>
    </property>
    <property name="openExternalLinks">
     <bool>true</bool>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::LinksAccessibleByMouse</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
